src/components/NavBarVertical/style.less

```diff
@@ -3,29 +3,21 @@
 
 .navbar-vertical {
   border-right: 1px solid rgba(51, 51, 51, 0.2);
+  color: #333;
```

Comment:
You can use the `@black` variable in https://github.com/acmucsd/membership-portal-ui/blob/4f2a620c01ae56152661f58d9063fd589ab120e5/src/styles/vars.less#L11